os/arch/arm/src/stm32l4/stm32l4_rtc.c

```diff
@@ -91,6 +91,9 @@
 
 #define RTC_ALRMR_ENABLE              (0x00000000)
 
+#define rtcinfo(...)
+#define rtcerr(...)
```

Comment:
Are these defines needed ?

Reply:
yes they are needed. Used in stm32l4 rtc driver without any definition. This resolved the build error for the same.